src/main/java/com/ericsson/ei/controller/FailedNotificationsController.java

```diff
@@ -0,0 +1,32 @@
+
+package com.ericsson.ei.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+
+/**
+ * This class searches for failed notifications in the database.
+ * (Generated with springmvc-raml-parser v.2.0.4)
+ *
+ */
+@RestController
+@Validated
+@RequestMapping(value = "/failed-notifications", produces = "application/json")
+public interface FailedNotificationsController {
```

Comment:
Which one of these files are generated by RAML, and which has been just renamed according to the Git history? Try cleaning both controllers and run the raml generation to see which one it generates.

Reply:
It seems that "s" is truncated by the RAML and therefore, we cannot generate plurals. In my previous pull request "s" was added , either by RAML (I dont know??) or I added manually (I am not sure).  Anyway I have removed the plural case.